nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -73,9 +73,38 @@ When NiFi first starts up, the following files and directories are created:
 
 See the <<system_properties>> section of this guide for more information about configuring NiFi repositories and configuration files.
 
+== Port Configuration
+
+=== NiFi
+The following table lists the default ports used by NiFi and the corresponding property in the _nifi.properties_ file.
+
+[options="header,footer"]
+|==================================================================================================================================================
+| Function                        |  Property                             | Default Value
+|HTTP Port                        | `nifi.web.http.port`                  | `8080`
+|HTTPS Port*                      | `nifi.web.https.port`                 | `9443`
+|Remote Input Socket Port*        | `nifi.remote.input.socket.port`       | `10443`
+|Cluster Node Protocol Port*      | `nifi.cluster.node.protocol.port`     | `11443`
+|Cluster Node Load Balancing Port | `nifi.cluster.node.load.balance.port` | `6342`
+|Web HTTP Forwarding Port         | `nifi.web.http.port.forwarding`       | blank
+|==================================================================================================================================================
+
+NOTE: The ports marked with an asterisk (*) have property values that are blank by default in _nifi.properties_.  The values shown in the table are the default values for these ports when <<tls_generation_toolkit>> is used to generate _nifi.properties_ for a secured NiFi instance.  The default Certificate Authority Port used by TLS Toolkit is `8443`.
```

Comment:
```suggestion
NOTE: The ports marked with an asterisk (*) have property values that are empty by default in _nifi.properties_.  The values shown in the table are the default values for these ports when <<tls_generation_toolkit>> is used to generate _nifi.properties_ for a secured NiFi instance.  The default Certificate Authority Port used by TLS Toolkit is `8443`.
```

Reply:
Similar to comment above about "blank", but happy to make this change since it reads better.

Reply:
`empty` is probably not correct here.  It could imply an empty string, which would not be the case for a property set like `nifi.web.http.port.forwarding=` as it is by default in nifi.properties.  `blank` is probably the better option to use in the descriptions, but maybe _`null`_ or _`none`_ in the port lists?

Reply:
Yup, will make _none_ in the tables.